test/features/support/ui_authorization_authenticate.js

```diff
@@ -0,0 +1,146 @@
+const pactum = require("pactum");
+const { Given, When, Then, Before, After } = require("@cucumber/cucumber");
+const {
+  localhost,
+  requestTime,
+  validTransactionId,
+  validIndividualId,
+} = require("./helpers/helpers");
+
+let specUIAuthAuthenticate;
+let invalidTransactionId;
+let invalidIndividualId;
+
+const baseUrl = `${localhost}authorization/authenticate`;
+
+const responseSchema = {
+  type: "object",
+  properties: {
+    responseTime: {
+      type: "string",
+    },
+    response: {
+      type: "object",
+      properties: {
+        transactionId: {
+          type: "string",
+        },
+      },
+      additionalProperties: false,
+    },
+    errors: {},
+  },
+  additionalProperties: false,
+};
+
+const requestFunction = (transactionId, individualId) =>
+  specUIAuthAuthenticate.post(baseUrl).withBody({
```

Comment:
The same problem as i mentioned in https://github.com/GovStackWorkingGroup/bb-identity/pull/17
Even if we send empty body, all tests will succeed.